### DISCUSSION TITLE
Not Supported for MAC Version 10.10.5

### DISCUSSION THREAD

**Discussion by pankaj-inbetween** [dbeaver/dbeaver#220] [Q&A] [answered]
The latest version Version 3.6.0 is not supported for mac version 10.10.5.

Can you please look it and help to resolve it ?

**Comment by serge-rider** [comment]
Please check #95 issue. Isn't it the same problem?
If not then please report more details.

**Comment by pankaj-inbetween** [comment, marked as answer]
I am closing this issue after changing few security thing in my mac system.
